pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -313,10 +322,24 @@ def export_testcase(
     os.makedirs(out_dir, exist_ok=True)
     if isinstance(args, torch.Tensor):
         args = args,
+        
+    def has_kwargs_in_args(args: tuple) -> bool:
+        return len(args) >= 1 and isinstance(args[-1], dict)
+
+    if has_kwargs_in_args(args):
+        assert "input_names" in kwargs, 'export_testcase needs explicit "input_names" when exporting with kwargs'
+        named_args_list = list(args[:-1])
+        for key, tensor in args[-1].items():
+            assert isinstance(tensor, torch.Tensor)
+            named_args_list.append(tensor)
```

Comment:
I feel implicit conversion seems to be useful for python scalar inputs
```suggestion
            if not isinstance(tensor, torch.Tensor):
                tensor = torch.tensor(tensor)
            named_args_list.append(tensor)
```

Reply:
I don't fully understand current behavior of export_testcase when a python scalar is given.
In export with testcase, we need to ensure that the number of onnx inputs and the number of testcase's input pb tensors match. If we implicitly create a tensor, doesn't it break this constraint?

I think it'll be safer if we just accept explicit tensor in such cases.

Reply:
OK. Your decision seems to be more reasonable! Thank you!